src/models.py

```diff
@@ -0,0 +1,67 @@
+import datetime
+from sqlalchemy import Column, Integer, String, DateTime, ForeignKey, BigInteger, DECIMAL
+from sqlalchemy.orm import relationship, backref
+from sqlalchemy.ext.declarative import declarative_base
+
+Base = declarative_base()
+
+# class Dummy(Base):
+#    __tablename__ = "dummy"
+#    id = Column(Integer, primary_key=True, index=True)
+#    dummy = Column(String(250),)
+
+
+class Currency(Base):
+    __tablename__ = "currency"
+
+    id = Column(Integer, primary_key=True, index=True)
+    symbol = Column(String(4), nullable=False, unique=True)
+    name = Column(String(20), nullable=False)
+    precision = Column(Integer)
+
+    created_at = Column(DateTime(timezone=True), default=datetime.datetime.utcnow)
+    upated_at = Column(DateTime(timezone=True), default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow)
+
+
+class Exchange(Base):
+    __tablename__ = "exchange"
+
+    id = Column(Integer, primary_key=True, index=True)
+    code = Column(String(20), nullable=False, unique=True)
+    name = Column(String(20), nullable=False)
+
+    created_at = Column(DateTime(timezone=True), default=datetime.datetime.utcnow)
+    upated_at = Column(DateTime(timezone=True), default=datetime.datetime.utcnow, onupdate=datetime.datetime.utcnow)
+
+class CurrencyPair(Base):
+    __tablename__ = "currency_pair"
+
+    id = Column(Integer, primary_key=True, index=True)
+    exchange_id = Column(Integer, ForeignKey('exchange.id'))
+    currency_a_id = Column(Integer, ForeignKey('currency.id'))
+    currency_b_id = Column(Integer, ForeignKey('currency.id'))
+    symbol = Column(String(10), nullable=False, unique=True)
```

Comment:
Following the increase of currency symbol size increment, we should increase this one.
```suggestion
    symbol = Column(String(20), nullable=False, unique=True)
```

Additionally, it will be nice to include one more column to store the market type. Each exchange have multiple market types, and order operations will be different depending market type. For example we can't open a short order in spot market, only swap and future.

My suggestion is to use the [Enum](https://docs.sqlalchemy.org/en/14/core/type_basics.html#sqlalchemy.types.Enum) with _spot_, _swap_ and _future_ options
We can discuss how to implement it later, or I can an additional commit to include it.